com.modrinth.ModrinthApp.yml

```diff
@@ -0,0 +1,103 @@
+        url: https://webkitgtk.org/releases/webkitgtk-2.42.5.tar.xz
+        sha256: b64278c1f20b8cfdbfb5ff573c37d871aba74a1db26d9b39f74e8953fe61e749
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DPORT=GTK
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_MINIBROWSER=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_SOUP2=ON
+      - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - "*"
+
+  - name: openssl11
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=${FLATPAK_DEST}
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install_sw
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - "*.a"
+      - "*.la"
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-1.1.1w.tar.gz
+        sha256: cf3098950cb4d853ad95c0841f1f9c6d3dc102dccfcacd521d93925208b76ac8
+        x-checker-data:
+          type: anitya
+          project-id: 20333
+          url-template: https://www.openssl.org/source/openssl-$version.tar.gz
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  # TODO: compile from source (preferably before the launcher supports arm on linux)
+  - name: modrinth-app
+    buildsystem: simple
+    build-commands:
+      - chmod +x ModrinthApp.AppImage
+      - ./ModrinthApp.AppImage --appimage-extract
+
+      - mkdir -p /app/{bin,share}
+      - cp -r squashfs-root/usr/share/icons /app/share/icons
+      - find /app/share/icons -name 'modrinth-app.png' -exec rename modrinth-app com.modrinth.ModrinthApp '{}' \;
+      - install -Dm755 squashfs-root/usr/bin/modrinth-app /app/bin/modrinth-app
+      - install -Dm644 com.modrinth.ModrinthApp.metainfo.xml /app/share/metainfo/com.modrinth.ModrinthApp.metainfo.xml
+      - install -Dm644 com.modrinth.ModrinthApp.desktop /app/share/applications/com.modrinth.ModrinthApp.desktop
+
+    sources:
+      - type: file
+        path: com.modrinth.ModrinthApp.metainfo.xml
+
+      - type: file
+        path: com.modrinth.ModrinthApp.desktop
+
+      - type: file
+        url: https://launcher-files.modrinth.com/versions/0.6.3/linux/modrinth-app_0.6.3_amd64.AppImage
```

Comment:
so how does that work when building the aarch64 package?

why not building from source?

Reply:
> so how does that work when building the aarch64 package?

it doesn't. the launcher doesn't support aarch64 on linux anyways

> why not building from source?

https://github.com/flatpak/flatpak-builder-tools/issues/383

Reply:
you mean you can't help with that problem either? I thought this was a Tauri app.

Reply:
> you mean you can't help with that problem either?

if i had the time, yeah i would love to help out. i just don't at the moment

> I thought this was a Tauri app.

it is, and like most tauri apps uses node for the frontend; in this case, upstream uses pnpm. again though, the launcher doesn't support anything besides x86_64 on linux anyways so this doesn't matter for now